Source/WebCore/platform/graphics/ca/cocoa/PlatformCALayerCocoa.mm

```diff
@@ -814,6 +814,33 @@ - (void)setOwner:(WebCore::PlatformCALayer*)owner
     updateContentsFormat();
 }
 
+void PlatformCALayerCocoa::setContentsActive(bool active)
+{
+    if (active != m_contentsActive) {
+        m_contentsActive = active;
+        BEGIN_BLOCK_OBJC_EXCEPTIONS
+        auto contentsFormat = this->contentsFormat();
+#if ENABLE(PIXEL_FORMAT_RGBA16F)
+        if (contentsFormat == ContentsFormat::RGBA16F) {
```

Comment:
Nit: `contentsFormat` is outside the #if block, so it could look "unused" on some platforms. Also it's only used once, in the `if` comparison. So I think you could lose it, e.g.: `if (contentsFormat() == ContentsFormat::RGBA16F)`.
(Unless there's another upcoming patch that will use it again?)